### ISSUE TITLE
Answer Jackie's question about what is allowed in fields for study profiles template

### ISSUE THREAD

**Issue by dcarsey** [RENCI/ctmd#438]
HI Dawn,

Not exactly I have created a more detailed instructions for all the fields that get uploaded, not just the upload instructions. On the attached file can you tell me if the items in red are programmatically defined or are they text fields?

For example under randomization type we have a number of options. Would it be possible technically to enter Jackie for example in the randomization type? I’m not clear if the options are something we are asking people to use or if they are required by the system.

So basically what is the type of field (text, date, numeric) and then things like field length, if numeric is it an integer or are decimals allowed, is there a specification that I could look at? One of the TICs is going to program a report from their system to match what we upload so that is why I am asking.

Jackie

**Comment by dcarsey** [comment]
Shruti pulled together this information about what the fields are set to accept:

[https://www.dropbox.com/s/7bc5pztz975oh3x/csvUploads_datatype%20response%20to%205-12-2021%20questions.docx?dl=0](url)
